### DISCUSSION TITLE
报了无法打开，因为无法验证开发者的错误

### DISCUSSION THREAD

**Discussion by ironsuperdev** [altairwei/WizNotePlus#251] [General]
<img width="260" alt="image" src="https://github.com/altairwei/WizNotePlus/assets/34617951/da90a5cb-9116-430a-87e2-e739b1c68a84">

先谢谢开发者，同为为知笔记的深度用户，觉得新版本的X不好用，所以四处在找解决方案

把最新的dmg考到应用后，无法打开，会报错，请问有解决办法吗？SIP？

**Comment by altairwei** [comment]
这是 macOS 的安全机制。我没有 Apple 开发者账号，没办法签名。你需要去 macOS 设置中心，允许这个应用。
